blimp-live-widgets/src/components/Clock/index.js

```diff
@@ -1,6 +1,7 @@
 import React, { Component } from 'react'
 import PropTypes from 'prop-types'
 import Styles from './styles'
```

Comment:
Why are there two style imports? Unless you think it's necessary, can you merge these?

Reply:
It's necessary because the old one is javascript code (for the analog) and the new one is CSS for the digital clock

Reply:
Maybe rename the imports then?